src/main/java/com/crowdin/util/FileUtil.java

```diff
@@ -52,6 +55,54 @@ public static VirtualFile getSourceFile(VirtualFile baseDir, String fileName) {
         return null;
     }
 
+    public static VirtualFile getBaseDir(VirtualFile file, String relativePath) {
+        VirtualFile dir = file;
+        for (int i = relativePath.split("/").length; i > 0; i--) {
```

Comment:
Maybe use file separator instead of `/` because it may lead to incorrect behaviour on a different OS

Reply:
There is little dependency on OS. VirtualFile always returns "/" as path separator. There may be more problems with using Path.separator than "/". And in this case, I replaced it with "[\\\\/]"